### ISSUE TITLE
Add Russian localization

### ISSUE THREAD

**Issue by AblakatovMikhail** [ti-assistant/ti-assistant#53]
Hello @SephHawkins !👋

First of all, I'd like to thank you for the work you've put into this app. We've run a couple of games using it and it's been a blast so far.

The only thing that kept interfering with the pace is the mismatch in the names of agendas, objectives and other cards. I've started working on a Russian localization and would like to check with you if you would be interested in adding this to your project. It should take me approx. 2 to 4 weeks to finish the translation.

_Cheers,
Mikhail._

**Comment by SephHawkins** [comment]
Hi @AblakatovMikhail!

Glad to hear that you've been using it and it's been going well.

I'd be more than happy to add a Russian translation to the site. No rush on the timeline, just let me know when you're done or if there's anything I can do to help. As a question, do you know what font the Russian version of TI uses for the titles of cards (and other things)? The English version uses Slider, but Slider doesn't appear to have support for Cyrillic characters, so I'll need to find the right fallback font.

Cheers!

**Comment by SanVerrigo** [comment]
Also, we don't have russian translation for TE yet, so is there a way to somehow mark strings that represent items from different expansions?

**Comment by SephHawkins** [comment]
Yes, that's correct. It looks like the column has already been added, so just add the translations there. Once it's translated enough, I'll add it to the site. 

Re: Thunder's Edge, I'll add a column showing which expansion each row is for.

**Comment by SephHawkins** [comment]
Ok - I've added a column for the expansion. It also has a filter, so you can filter by different expansions to hide the rows that you're not able to translate yet.

**Comment by SanVerrigo** [comment]
Thank you!
But I noticed that expansion's row is missing for Attachments. What I recognized from PoK - I added label there. But other attachments I do not know from which expansion.

<img width="1132" height="753" alt="Image" src="https://github.com/user-attachments/assets/6357696c-4268-4cc8-83ab-5129fabc4fc8" />

**Comment by SephHawkins** [comment]
Thanks for the catch - there were a few other rows missing expansions as well. I've added them in, and double checked that every row has an expansion.

Let me know if there's anything else you find. If not, I'll add the translation to the site once it's got _most_ of the strings translated.

**Comment by SanVerrigo** [comment]
Hello!
We every right-now translated strings in russian to your sheet (this includes in-game content of Base Game + PoK + Codices I-III and UI elements).
Also, I wrote a script for Apps Script to parse table to JSON file. Here it is:
```js
function myFunction() {
  const fileName = "ru.json" // setup name of resulting file
  const columnNameTranslation = "Russian"; // setup column title

  // actual code
  const activeSS = SpreadsheetApp.getActiveSpreadsheet();
  const columnNameId = "Message ID";
  const columnNameEn = "English";
  const columnNameDescription = "Description";

  const activeS = activeSS.getActiveSheet();
  const data = activeS.getDataRange().getValues();
  const firstRow = data[0];
  const messageIdIndex = firstRow.findIndex((value, index, obj) => value == columnNameId);
  const russianIndex = firstRow.findIndex((value, index, obj) => value == columnNameTranslation);
  const englishIndex = firstRow.findIndex((value, index, obj) => value == columnNameEn);
  const descriptionIndex = firstRow.findIndex((value, index, obj) => value == columnNameDescription);


  const dynamicDict = {};
  for(let i = 1; i < data.length; i++) {
    const activeRow = data[i];
    if (activeRow[descriptionIndex] == "") {
      continue;
    }
    let message = activeRow[russianIndex];
    if (message == "") {
      message = activeRow[englishIndex];
    }

    const entry = {
      defaultMessage: message,
      description: activeRow[descriptionIndex]
    };
    dynamicDict[activeRow[messageIdIndex]] = entry;
  }
  const jsonString = JSON.stringify(dynamicDict)
  
  const file = DriveApp.createFile(fileName, jsonString);
}
```

Also we noticed that nor your `en.json` file neither your sheet does not contain next things:
- Planets (all of them)
- UI Strings:
  - Fleet Pool Tokens
  - Support for the Throne
  - Alliance
  - Unowned Relics

But all in all this works! We checked this out on our VM and it wasn't bad.

Also we noticed that site's font do not work well with cyryllic text. So we found that in english next fonts are used:
- Titles + Abilities + Numbers - Handel Gothic TL
- "ACTION" - Myriad Pro Bold Italic
- Other text - Myriad Pro Bold/Semibold

In cyryllic next fonts are used:
- Titles + Abilities + Numbers - RussoOne
- Other text - Myriad Pro Bold/Semibold

RussoOne can be found in attached `Fonts.zip`.

Thank you for your work and for your responses!
[Fonts.zip](https://github.com/user-attachments/files/25129934/Fonts.zip)
